src/content/consul/product-landing.json

```
@@ -29,7 +29,7 @@
 			{
 				"icon": "hcp",
 				"text": "HCP Consul Quick Start",
-				"url": "/consul/tutorials/cloud-get-started/get-started"
+				"url": "/consul/tutorials/get-started-hcp/get-started"
```

I'm getting a 404 at [/consul/tutorials/get-started-hcp/get-started](https://dev-portal-git-kendallstrautman-patch-1-hashicorp.vercel.app/consul/tutorials/get-started-hcp/get-started).

This might not be the correct link, mostly a guess, but if it is then I figured a lil suggestion block might be helpful:

```suggestion
				"url": "/consul/tutorials/get-started-hcp/hcp-gs-deploy"
```

---

Ahh I checked the collection but didn't check the tutorial reference! ty 😄